### ISSUE TITLE
Steam Losses table Zero values

### ISSUE THREAD

**Issue by koay9f** [ORNL-AMO/AMO-Tools-Desktop#2851]
When the loss = 0 the line doesn't show up on the table.

This is good for when the loss *shouldn't be there* (a turbine loss for a turbine that isn't present), but not good if there is a 100% eff turbine or a turbine with zero flow.  may want to rethink how it works

**Comment by koay9f** [comment]
[https://app.zenhub.com/files/80439269/24ebc6d9-8828-4851-a942-88a429c19737/download](https://app.zenhub.com/files/80439269/24ebc6d9-8828-4851-a942-88a429c19737/download)

I think this is only a concern for turbines.  Can you send a list (in slack, I won't notice it here) of any of the lines in that table that can "disappear" and what condition makes them appear/disappear?

An example is in the ppt above. Or I can send you an assessment via slack if you want.  just let me know

**Comment by koay9f** [comment]
I think this is causing the Sankey to crash.  May need to pull in Nick.

Below is an example.  you will need to change from .txt to .json

[https://app.zenhub.com/files/80439269/0ca983c4-2271-4d52-beaa-336774aff918/download](https://app.zenhub.com/files/80439269/0ca983c4-2271-4d52-beaa-336774aff918/download)
